docs/getting-started/using-sqlfmt.mdx

```diff
@@ -42,4 +49,40 @@ sqlfmt --diff .
 sqlfmt can also format code passed through standard input (`stdin`) by passing `-` as the files argument. The formatted code will be output to `stdout` (all other output from sqlfmt is routed to `stderr`):
 ```bash
 echo "select 1" | sqlfmt -
-```
\ No newline at end of file
+```
+</TabItem>
+<TabItem value="docker" label="Docker">
+
+sqlfmt is also available as a Dockerized container. This allows for formatting without having Python installed. It currently does not support the `check` and `diff` functionality.
```

Comment:
This part isn't true. You can pass a command into `docker run` to override the default command. So this will run sqlfmt in check mode (and you can pass other command line args this way):

```bash
docker run -v $(pwd):/src ghcr.io/tconbeer/sqlfmt:latest sqlfmt . --check --line-length 100
```

Reply:
You learn something new every day. Will update it.